src/main/java/network/brightspots/rcv/CastVoteRecord.java

```diff
@@ -59,13 +59,13 @@
   // a set is used to handle overvotes
   SortedMap<Integer, Set<String>> rankToCandidateIds;
   // contest associated with this CVR
-  private Integer contestId;
+  Integer contestId;
   // tabulatorId parsed from Dominion CVR data
-  private Integer tabulatorId;
+  Integer tabulatorId;
   // batchId parsed from Dominion CVR data
-  private Integer batchId;
+  Integer batchId;
   // ballotTypeId parsed from Dominion CVR data
-  private Integer ballotTypeId;
+  Integer ballotTypeId;
```

Comment:
Please keep all these private (including `rankToCandidateIds`) and create getters instead (like we've done in other classes). I believe there's even a shortcut to do this in IntelliJ with the push of a button.